### ISSUE TITLE
301 and trailing slash

### ISSUE THREAD

**Issue by paulgirard** [medialab/website#505]
all pages on the website exists only with a trailing slash.
But all internal links are without trailing slash.
Result : all internal links first generates a 301 redirect from nginx to consult the page with a trailing slash.

Feels like not optimal to me.

## Solution 1 : 
why doesn't pages exist without trailing slash ?
Can we fix this ? 
## Solution 2 : 
Add trailing slash to internal link ? 
## Solution 3: 
ignore this issue which only people testing crawlers on the website like me might see. ever.

**Comment by boogheta** [comment]
I would be more in favor of removing the trailing slash from real urls, so no change to internal links, but a change in wilson's way of building
(and add a redirection from slash ones to no slash of course)

**Comment by Yomguithereal** [comment]
So the same but in reverse where we don't have trailing slashes :) ?

**Comment by boogheta** [comment]
yes and no, I said "no change to internal links", so in this case we wouldn't have to follow a redirec every time we click a link ;)
